docs/docs/migration.md

```diff
@@ -4,6 +4,8 @@ If you have previously used the [Multitarget Application Archive Builder](https:
 
 #### Features that handled differently in the Cloud MTA Build Tool
 
+* For building an MTA project, the Cloud MTA build Tool uses `GNU Make` technology. Therefore, you should have `GNU Make` installed in your build environmnet. Please read the correponding sections for more information about [`GNU Make` installation](makefile.md) and [commands for building a project](usage.md#how-to-build-an-mta-archive-from-the-project-sources). 
```

Comment:
The Cloud MTA build Tool uses `GNU Make` technology for building MTA projects. Therefore, you should have `GNU Make` installed in your build environment. For more information, see the sections: [`GNU Make` installation](makefile.md) and [commands for building a project](usage.md#how-to-build-an-mta-archive-from-the-project-sources).

Reply:
Updated

Reply:
Thanks